examples/wearable/UIComponents/contents/presentation-views/list/main/icon-right.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+	<meta content="width=device-width, initial-scale=1, user-scalable=no" name="viewport" />
+	<title>
+		UI Components
+	</title>
+	<link href="../../../../lib/tau/wearable/theme/default/tau.css" rel="stylesheet" />
+	<link href="../../../../lib/tau/wearable/theme/default/tau.circle.css" rel="stylesheet" />
+	<link href="../../../../css/style.css" rel="stylesheet" />
+	<link href="../../../../css/style.circle.css"  rel="stylesheet" />
+	<script src="../../../../js/circle-helper.js">
+	</script>
```

Comment:
In my personal opinion, it's nice to see that the last tag is moved to shorten the line.

Reply:
Done